src/include/distributed/multi_router_executor.h

```diff
@@ -9,15 +9,37 @@
 #ifndef MULTI_ROUTER_EXECUTOR_H_
 #define MULTI_ROUTER_EXECUTOR_H_
 
+#include "c.h"
+
+#include "access/sdir.h"
 #include "distributed/multi_physical_planner.h"
 #include "executor/execdesc.h"
+#include "nodes/pg_list.h"
+
+/* maximum (textual) lengths of hostname and port */
+#define MAX_NODE_LENGTH 255
```

Comment:
So… this comment is wrong. But also I don't need this `#define` anymore. Removing.